src/patternfly/components/AlertGroup/examples/alert-group-example.hbs

```diff
@@ -0,0 +1,50 @@
+{{#> alert-group}}
+  {{#> alert-item}}
+    {{#> alert alert--modifier="pf-m-success" alert--attribute='aria-label="Success Notification"'}}
+      {{#> alert-icon alert-icon--success="true"}}
```

Comment:
@srambach taught me a cool thing today. If you checkout the [handlebars partials docs](https://handlebarsjs.com/partials.html), you'll see there are basic partials and block partials. Basic is when there is no content in the partial (you're just including a partial), and block is when you include content with the partial. So with....

```
{{#> alert-icon alert-icon--success="true"}}
{{/alert-icon}}
```

You can just write `{{> alert-icon alert-icon--success="true"}}`

\#sarahisagenius